contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -79,12 +88,17 @@ contract PixelDevsUkraineDonation is
         _setTokenURI(newTokenId, donationType);
     }
 
-    function withdraw() public onlyOwner {
-        payable(msg.sender).transfer(address(this).balance);
+    function withdraw() public onlyRole(WITHDRAW_ROLE) {
+        payable(withdrawWallet).transfer(address(this).balance);
     }
 
-    function switchContractState() public onlyOwner {
+    function switchContractState() public onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
Got a question, what is `contractState` meant for?

Reply:
It's mean to turn the contract on/off. So when we want to close it down, we could turn it off, then close down UI.

Reply:
I see, there is also a contract for that on OpenZeppelin that we can inherit from. Gives us the modifiers and necessary internal functions as needed to pause/unpause the contract.

Though just sharing that, could be useful for other use cases.

https://docs.openzeppelin.com/contracts/4.x/api/security#Pausable

cc @briangershon @goldzulu @rasmuscnielsen @itstargetconfirmed